xbmc/interfaces/json-rpc/SettingsOperations.cpp

```diff
@@ -684,6 +695,26 @@ bool CSettingsOperations::SerializeSettingAddon(const CSettingAddon* setting, CV
   return true;
 }
 
+bool CSettingsOperations::SerializeSettingDate(const CSettingDate* setting, CVariant &obj)
```

Comment:
follow up question though: is the xml really of any value here or could you use json on disk as well? i doubt many ever edit these by hand.